recipe/meta.yaml

```diff
@@ -38,14 +38,16 @@ package:
 
 source:
   url: https://github.com/kaldi-asr/kaldi/archive/{{ commit }}.tar.gz
-  sha256: 153ae41654046f1997c247d3328c17124745b70be090fe1ee73442160e7b08c7
+  sha256: aae30bb72ca2522eeec6582aaad092b4188fa17af33acb9a8d970419151edbd6
   patches:
-    - patches/0001-Update-cmake.patch
-    - patches/0002-Remove-Windows-executables.patch
+    #- patches/0001-Update-cmake.patch
+    #- patches/0002-Remove-Windows-executables.patch
```

Comment:
If we don't need these patches anymore, you can delete them IMO.

Reply:
Sure, sounds good, should I renumber the 0003 and 0004 patches to 0001/0002 or keep them as is for consistency?

Reply:
I don't really care about the numbering too much, though I do care about authorship information and an explanation is the commit message. This isn't currently present in your patch (and it's not a deal-breaker), but in case you have (or create) a branch locally where those commits have been applied, you could kill three birds with one stone by doing:
`git format-patch <tag_or_hash_of_version_we're_based_on>`

Reply:
Ok, regenerated the patches based on this, it adds more of them due to a commit for each version of OpenFst, but keeps the messages.